hugo/data/you_draw_it_karlsruhe_paris_data.json

```diff
@@ -37,132 +37,144 @@
       "value": 2558.452
     },
     {
-      "year": 2020.0,
+      "year": 2015,
```

Comment:
Das Jahr 2015 ist hier doppelt.

Reply:
Siehe Kommentar oben, diese Datei wird in Zukunft gar nicht mehr gebraucht.